src/katsdpcontroller/consul.py

```diff
@@ -27,6 +28,10 @@
 
 CONSUL_PORT = 8500
 CONSUL_URL = yarl.URL.build(scheme="http", host=LOCALHOST, port=CONSUL_PORT)
+# Usually needs must less than this, but registering a service requires
+# storing the record on disk in the consul catalogue, and things like VM
+# backups can make that unusually slow.
+CONSUL_TIMEOUT = 20
```

Comment:
To confirm, your PR blurb says
> Increase the timeout from 5 to 15s

Just want to make sure this value is correct/as intended.

Reply:
Ah, I was dithering over whether to make it 15s or 20s. I'll change it to 15s: now that you've cut down the qualification testing time, we shouldn't encounter the backup stall any more, so we don't need to be too conservative.